snapcraft.yaml

```diff
@@ -1375,6 +1376,16 @@ parts:
 
       patch -p1 "${CRAFT_PART_INSTALL}/snap/lxd/current/lxcfs/lxc.mount.hook" < "${CRAFT_PROJECT_DIR}/patches/lxcfs-0001-hook.patch"
 
+  uefivars:
```

Comment:
I see there is only one release of python-uefivars - but should we bind it this specific tag (`1.0.0`), or is latest ok?

Reply:
we should indeed or a git sha

Reply:
Sure, I'll do version pinning. My bad. Thanks!

Reply:
@mihalicyn please can you do the version pinning and i can merge this, ta

Reply:
Have done!